providers/websockets/kucoin/hooks.go

```diff
@@ -0,0 +1,159 @@
+package kucoin
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/goccy/go-json"
```

Comment:
So here we are using this package, but elsewhere we always use the std package.  Is there a reason for this?

Reply:
that is an accident lol